build_exe.bat

```diff
@@ -0,0 +1,11 @@
+@echo off
+python --version | find /i "Python 3.4" > nul
+echo -----
+if errorlevel 1 (
+	echo Warning : Please use Python 3.4.x
+) else (
+	echo Python version check done!
+)
+echo -----
+python -m PyInstaller NAFAS.spec
+pause
```

Comment:
According to last comment there is another newline miss here too.

Reply:
Solved in https://github.com/sepandhaghighi/nafas/pull/10/commits/1c8c9c154b68c2094e5e9327b11ad7aa263ff325